serokell-style.md

```diff
@@ -93,10 +102,16 @@ as constructor name (also applies to `newtype`).
 data User = User Int String
 ```
 
-Field name for `newtype` should start with `get` prefix followed by type name.
+Field name for `newtype` should start with `un` or `run` prefix followed by type name.
+
+* `run` for wrappers with monadic semantic.
+* `un` for wrappers introduced for type safety.
```

Comment:
I see no reason to use `un` where `get` is the default. `run/get` separation is good.

Reply:
You probably didn't read Reddit discussion.

If I have

```haskell
newtype ProjectId = ProjectId { getProjectId :: Text }
```

then it's awkward to give a name to function with type `MonadDB m => m ProjectId`. Can be `queryProjectId` but it's longer than `getProjectId`. `un` is shortage for `unwrapping` (because usually you introduce newtypes for wrapping functions).

Reply:
Ok.

Reply:
`un` also is a common practice, so I feel it should be allowed as well.

Reply:
I like `un`.